index.bs

```diff
@@ -4377,6 +4378,38 @@ navigated</dfn> steps given |context| and |navigation status|:
 
 </div>
 
+#### The browsingContext.historyUpdated Event #### {#event-browsingContext-historyUpdated}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.HistoryUpdated = (
+         method: "browsingContext.historyUpdated",
+         params: {
+            url: text,
+         }
+        )
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi history updated</dfn> steps given |navigable|:
+
+1. Let |body| be a [=/map=] matching the
+    <code>browsingContext.HistoryUpdated</code> production, with the
+    <code>url</code> field set to |navigable|'s [=active browsing context=]'s [=active document=]'s <a spec=DOM>URL</a>.
```

Comment:
If `url` is text, I imagine this should rather be 

>  url field set to the result of running the [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer), given navigable's active document’s [URL](https://dom.spec.whatwg.org/#concept-document-url).

Or something similar?

Reply:
updated